torch/ao/quantization/fx/prepare.py

```diff
@@ -523,7 +523,7 @@ def _get_output_act_obs_or_fq_ctr(
     argument in quantized graph will match what is specified by the qconfig
     """
     assert isinstance(arg, Node)
-    if "quantization_annotation" in arg.meta:
+    if "quantization_annotation" in arg.meta and arg.meta["quantization_annotation"].output_qspec is not None:
```

Comment:
is this needed?

Reply:
Yes, I think so. Otherwise a `None` object is return by `_get_output_act_obs_or_fq_ctr` and it fails the check of `_needs_obs_or_fq` in https://github.com/pytorch/pytorch/blob/8243abc84ada1a7c8153d7257b60ba4ffdc7b7cb/torch/ao/quantization/_pt2e/prepare.py#L66-L74. So there will be no observer inserted at input of `maxpool2d`.

Reply:
I see, this logic is changed again in the next PR (not yet in oss), please wait a bit

Reply:
Thanks, will wait for next PRs. Please let me know when to do the rebase.